package.json

```diff
@@ -17,7 +17,8 @@
     "prepare": "husky install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:headed": "cypress run --headed"
+    "cypress:headed": "cypress run --headed",
+    "update-types": "cross-env dotenv -e .env -- bash -c 'npx supabase gen types --lang=typescript --project-id $PROJECT_ID --schema public > supabase/types.ts'"
```

Comment:
Without a migration file or a way to setup a DB programmatically, if a new contributor wanted to use their own DB they'd need to root around your code/types and manually create it first.

Failing that they'd erase your types and the logic would fail as the DB would not be ready to accept the data. They'd also have to create RLS policies too, which would typically be a part of the migration file.

What is your solution to this problem?